packages/data-stores/src/plans/constants.ts

```diff
@@ -61,3 +94,16 @@ export const PERIOD_LIST = [
 	PLAN_BIENNIAL_PERIOD,
 	PLAN_TRIENNIAL_PERIOD,
 ] as const;
+
+// Non plan-specific constants
+export const TERM_MONTHLY = 'TERM_MONTHLY';
+export const TERM_ANNUALLY = 'TERM_ANNUALLY';
+export const TERM_BIENNIALLY = 'TERM_BIENNIALLY';
+export const TERM_TRIENNIALLY = 'TERM_TRIENNIALLY';
+
+export const TERMS_LIST = [
+	TERM_MONTHLY,
+	TERM_ANNUALLY,
+	TERM_BIENNIALLY,
+	TERM_TRIENNIALLY,
+] as const;
```

Comment:
These should probably move down the line to some other central/shared location. Presumably term constants can be shared by the [products store](https://github.com/Automattic/wp-calypso/tree/trunk/packages/data-stores/src/products-list), as we build out that part more. We have "shared-types" in the root, so "shared-something" might make sense.